### PR TITLE
Improve bulk data example and notes 

### PR DESCRIPTION
**Pull request by kstetsyuk**

### CODE REVIEW

File: Python/facility_data_demo.py [hunk 1]
```diff
@@ -1,35 +1,52 @@
...
+
 # api parameters for the streaming facility/attributes endpoint
 parameters = {
     'api_key': API_KEY,
-    'year': 2020,
+    'year': '2020|2021',
     'unitFuelType': 'Coal|Natural Gas'
 }
 
+if len(parameters['year']) > 4:
```
Comment (j-tafoya) [inline Python/facility_data_demo.py:29]: This isn't parsing the 2020|2021 as a list, but instead counting the length of the string. You may want to check this another way.

File: R/account_data_demo.R [hunk 2]
```diff
@@ -2,29 +2,47 @@ library(httr)
 library(htmltools)
 library(jsonlite)
```
Comment (j-tafoya) [inline R/account_data_demo.R:4]: Need `library(dplyr)`

**Review by j-tafoya — CHANGES_REQUESTED** [review]
Just a few minor changes needed, some of which are "nice-to-haves" so feel free to ignore those

**Comment by kstetsyuk** [comment]
@j-tafoya I will get on these - they all sound good to me. Thank you for suggestions and extra eyes!!

**Comment by kstetsyuk** [comment]
@j-tafoya Can you take a look at this when you can? Thanks!

**Review by j-tafoya — COMMENTED** [review]
Looks good! Just a couple of package imports needed and it's good to go! Thanks for all your great work on this! This will be super helpful for users!

**Review by j-tafoya — APPROVED** [review]
Perfect! Thanks!

**Pull request merged**